<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="6" total="7" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-06-11T21:52:38 CEST" name="Default suite" finished-at="2021-06-11T21:53:11 CEST" duration-ms="33553">
    <groups>
    </groups>
    <test started-at="2021-06-11T21:52:38 CEST" name="Default test" finished-at="2021-06-11T21:53:11 CEST" duration-ms="33553">
      <class name="automationPracticeTests.MyAddressesTest">
        <test-method is-config="true" signature="preSihTestova()[pri:0, instance:automationPracticeTests.MyAddressesTest@3514a4c0]" started-at="2021-06-11T21:52:38 CEST" name="preSihTestova" finished-at="2021-06-11T21:52:46 CEST" duration-ms="8210" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preSihTestova -->
        <test-method is-config="true" signature="preSvakogTesta()[pri:0, instance:automationPracticeTests.MyAddressesTest@3514a4c0]" started-at="2021-06-11T21:52:46 CEST" name="preSvakogTesta" finished-at="2021-06-11T21:52:55 CEST" duration-ms="8310" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preSvakogTesta -->
        <test-method signature="updateMainAddress()[pri:0, instance:automationPracticeTests.MyAddressesTest@3514a4c0]" started-at="2021-06-11T21:52:55 CEST" name="updateMainAddress" finished-at="2021-06-11T21:53:09 CEST" duration-ms="14113" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Hajduk Veljkova 5] but found [YOUR ADDRESSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Hajduk Veljkova 5] but found [YOUR ADDRESSES]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at automationPracticeTests.MyAddressesTest.updateMainAddress(MyAddressesTest.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateMainAddress -->
        <test-method is-config="true" signature="posleSvakogTesta()[pri:0, instance:automationPracticeTests.MyAddressesTest@3514a4c0]" started-at="2021-06-11T21:53:09 CEST" name="posleSvakogTesta" finished-at="2021-06-11T21:53:11 CEST" duration-ms="2242" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- posleSvakogTesta -->
        <test-method is-config="true" signature="posleSihTestova()[pri:0, instance:automationPracticeTests.MyAddressesTest@3514a4c0]" started-at="2021-06-11T21:53:11 CEST" name="posleSihTestova" finished-at="2021-06-11T21:53:11 CEST" duration-ms="423" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- posleSihTestova -->
      </class> <!-- automationPracticeTests.MyAddressesTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
